exe/sdr

```diff
@@ -62,6 +62,11 @@ subcommands = {
       options[:source_id] = source_id
     end
 
+    opts.on('--viewing-direction DIRECTION', 'The viewing direction (if a book). ' \
```

Comment:
`s/if/of/`

Reply:
well, it doesn't make sense to set it on a 3d object for example.

Reply:
In fact, it's ignored in DSA if the type isn't book.

Reply:
Oh, I thought that was a typo. I getcha now.